Sources/FioriSwiftUICore/_FioriStyles/StepProgressIndicatorStyle.fiori.swift

```diff
@@ -0,0 +1,217 @@
+import FioriThemeManager
+import Foundation
+import SwiftUI
+
+/// :nodoc:
+public extension StepProgressIndicator {
+    init(selection: Binding<String>,
+         @ViewBuilder title: () -> any View = { EmptyView() },
+         @ViewBuilder action: () -> any View = { EmptyView() },
+         @ViewBuilder cancelAction: () -> any View = { FioriButton { _ in Text("Cancel".localizedFioriString()) } },
+         @IndexedViewBuilder steps: () -> any IndexedViewContainer = { EmptyView() })
+    {
+        self.init(title: title, action: action, cancelAction: cancelAction, selection: selection, steps: steps)
+    }
+
+    /// Convenience initialization for default step progress indicator.
+    /// - Parameters:
+    ///   - selection: A binding string for selected step id.
+    ///   - stepItems: An array of `StepItem` for default steps generation.
+    ///   - title: Title for current step displayed on steps top-left .
```

Comment:
```suggestion
    ///   - title: Title for current step displayed on the top leading side of the step progress indicator .
```
Usually we don't use the word left or right, since our control supports RTL layout